worlds/undertale/__init__.py

```diff
@@ -0,0 +1,228 @@
+from .Items import UndertaleItem, item_table, required_armor, required_weapons, non_key_items, key_items, \
+    junk_weights_all, plot_items, junk_weights_neutral, junk_weights_pacifist, junk_weights_genocide
+from .Locations import UndertaleAdvancement, advancement_table, exclusion_table
+from .Regions import undertale_regions, link_undertale_areas
+from .Rules import set_rules, set_completion_rules
+from worlds.generic.Rules import exclusion_rules
+from BaseClasses import Region, Entrance, Tutorial, Item
+from .Options import undertale_options
+from worlds.AutoWorld import World, WebWorld
+from worlds.LauncherComponents import Component, components, Type
+from multiprocessing import Process
+
+
+def run_client():
```

Comment:
This will probably need some revamping in the future, once we figure out a proper and stable api to embed clients.